hints.yaml

```
@@ -14,7 +14,7 @@ Sky - Gossip Stone in Bamboo Island:
   textfile: 004-Object
   textindex: 11
   Need: Nothing
-Thunderhead - Gossip Stone on Bug Heaven:
+Thunderhead - Gossip Stone on Bug Rock:
```

```suggestion
Thunderhead - Gossip Stone near Bug Heaven:
```
It's actually not on the island, also I prefer Bug Heaven over Bug Rock